src/Sylius/Component/Translation/Model/AbstractTranslatable.php

```diff
@@ -19,15 +19,11 @@
 abstract class AbstractTranslatable implements TranslatableInterface
 {
     /**
-     * Translations.
-     *
      * @var TranslationInterface[]
```

Comment:
Ok, construct in abstract class is not the thing I woule expect :) Anyway, that should apply to `TranslatableInterface`.